lib/printer.js

```diff
@@ -1155,6 +1155,29 @@ function genericPrintNoParens(path, options, print) {
 
         return concat(parts);
 
+    case "TemplateElement":
+        return n.value.raw;
+
+    case "TemplateLiteral":
+        var expressions = path.map(print, "expressions");
+        var parts = [];
+        path.map(function(childPath) {
```

Comment:
I think it's appropriate to use `path.each` here, since you're not using the resulting array.